volatility3/framework/symbols/linux/extensions/__init__.py

```diff
@@ -13,12 +13,24 @@
 
 from volatility3.framework import constants, exceptions, objects, interfaces, symbols
 from volatility3.framework.renderers import conversion
-from volatility3.framework.constants.linux import SOCK_TYPES, SOCK_FAMILY
-from volatility3.framework.constants.linux import IP_PROTOCOLS, IPV6_PROTOCOLS
-from volatility3.framework.constants.linux import TCP_STATES, NETLINK_PROTOCOLS
-from volatility3.framework.constants.linux import ETH_PROTOCOLS, BLUETOOTH_STATES
-from volatility3.framework.constants.linux import BLUETOOTH_PROTOCOLS, SOCKET_STATES
-from volatility3.framework.constants.linux import CAPABILITIES, PT_FLAGS
+from volatility3.framework.constants.linux import (
```

Comment:
This might be easier to view as `from ...constants import linux` and then refer to `linux.SOCK_TYPES`, etc?  Seems like it might tidy the imports and keep with the coding style too...

Reply:
I agree, it was quite a mess. I attempted to improve it a bit with that change, but it really needed the changes you're suggesting. It's done now. 
I renamed the import to `linux_constants` since `linux` was already used for the `symbols` import. I believe this makes it clearer, as it identifies exactly what it is rather than just a bare `linux`.

Reply:
Yep, makes perfect sense, good call!  5:)